client/my-sites/guided-transfer/guided-transfer.jsx

```diff
@@ -86,7 +86,11 @@ export default React.createClass( {
 				{ this.props.isEligibleForGuidedTransfer
 					? <div className="guided-transfer__content">
 						{ hostInfo
-							? <HostCredentialsPage hostSlug={ this.props.hostSlug } hostInfo={ hostInfo } />
+							? <HostCredentialsPage
+								siteId={ this.props.siteId }
+								siteSlug={ this.props.siteSlug }
```

Comment:
Moved `siteSlug` in 11cbce3. We could probably do the same for `siteId` in the other components, but I'll leave that for another PR.